src/frontend/src/pages/HomePage/components/InfoBlock.tsx

```diff
@@ -0,0 +1,65 @@
+import { Grid, Typography, ListItem, Card, List, useTheme } from '@mui/material';
```

Comment:
You have a linting error which means you have imported/defined something that isn't being used. In this case it is the card component so you can just delete that from the imports